ee/debug/checkups/osquery_restarts.go

```diff
@@ -0,0 +1,62 @@
+package checkups
+
+import (
+	"context"
+	"errors"
+	"io"
+
+	"github.com/kolide/launcher/ee/agent/types"
+	"github.com/kolide/launcher/pkg/osquery/runtime/history"
+)
+
+type (
+	osqRestartCheckup struct {
+		k       types.Knapsack
+		status  Status
+		summary string
+		data    map[string]any
+	}
+)
+
+func (orc *osqRestartCheckup) Data() any             { return orc.data }
+func (orc *osqRestartCheckup) ExtraFileName() string { return "" }
+func (orc *osqRestartCheckup) Name() string          { return "Osquery Restarts" }
+func (orc *osqRestartCheckup) Status() Status        { return orc.status }
+func (orc *osqRestartCheckup) Summary() string       { return orc.summary }
+
+func (orc *osqRestartCheckup) Run(ctx context.Context, extraFH io.Writer) error {
+	orc.data = make(map[string]any)
+
+	restartHistory, err := history.GetHistory()
+	if err != nil && errors.Is(err, history.NoInstancesError{}) {
+		orc.status = Passing
```

Comment:
Consider whether this is Passing or Informational.

Reply:
Good catch, Informational does seem more appropriate here I will update